SilKit/source/services/orchestration/TimeSyncService.cpp

```diff
@@ -487,7 +489,14 @@ void TimeSyncService::ExecuteSimStep(std::chrono::nanoseconds timePoint, std::ch
     const auto waitingDurationMs = std::chrono::duration_cast<DoubleMSecs>(waitingDuration);
     const auto waitingDurationS = std::chrono::duration_cast<DoubleSecs>(waitingDuration);
 
-    Trace(_logger, "Starting next Simulation Task. Waiting time was: {}ms", waitingDurationMs.count());
+    {
```

Comment:
the explicit building of a loggermessage prevents optimizing it away.
Couldn't we just extent the `Logging::Debug(...)` template to expand an initializer list of tuples to LoggerMessages internally?
E.g. couldn't we make it to look like:
` Trace(_logger, "Starting next Simulation Task. Waiting time was: {}ms", waitingDurationMs.count(), {Logging::Keys::waitingTime, waitingDurationMs}, { Logging::Keys::virtualTimeNS, timePint});`
e.g. this would allow to not evaluate the arguments in case the loggers loglevel is not set, and also providing printing overloads for timepoints and durations along the way?